Fundamentals/Setup/Upgrading/version-specific.md

```diff
@@ -8,6 +8,49 @@ versionFrom: 7.0.0
 
 Follow the steps in the [general upgrade guide](general.md), then these additional instructions for the specific versions. (Remember that new config files are not mentioned because they are already covered in the general upgrade guide.)
 
+## Version 9 to version 10
+The upgrade path between Umbraco 9 and Umbraco 10 and be done directly by updating the Nuget references. Additional to that you will have to remove a couple of files from your solution.
+
+We updated the `Program` class in the `Program.cs` file to the following out of box:
+
+```cs
+ public class Program
+    {
+        public static void Main(string[] args)
+            => CreateHostBuilder(args)
+                .Build()
+                .Run();
+
+        public static IHostBuilder CreateHostBuilder(string[] args) =>
+            Host.CreateDefaultBuilder(args)
+                .ConfigureUmbracoDefaults()
+                .ConfigureWebHostDefaults(webBuilder =>
+                {
+                    webBuilder.UseStaticWebAssets();
+                    webBuilder.UseStartup<Startup>();
+                });
+    }
+```
+
+The calls to `ConfigureUmbracoDefaults` and `webBuilder.UseStaticWebAssets()` are new.
+
+Furthermore it is recommended to remove the following files and folders:
+
+- `/wwwroot/umbraco`
+- `/umbraco/PartialViewMacros`
+- `/umbraco/UmbracoBackOffice`
+- `/umbraco/UmbracoInstall`
+- `/umbraco/UmbracoWebsite`
+- `/umbraco/lang`
+
+All these files are now served directly from a Razor Class Library (Umbraco.Cms.StaticAssets) or as embedded resources.
+
+
+:::note
+Please note to upgrade to Umbraco 10, you need your database to be at least on Umbraco 8.18.
```

Comment:
```suggestion
To upgrade to Umbraco 10, you need your database to be at least on Umbraco 8.18.
```
Since it's already in a note tag we need not write Please note again 🙂